src/components/ContributionsEpic.tsx

```diff
@@ -81,45 +97,57 @@ export type EpicMetadata = {
 type Props = {
     content: EpicContent;
     metadata: EpicMetadata;
+    countryCode?: string;
 };
 
 type HighlightedProps = {
     highlighted: string[];
+    countryCode?: string;
 };
 
 type BodyProps = {
     paragraphs: string[];
     highlighted: string[];
+    countryCode?: string;
 };
 
-const Highlighted: React.FC<HighlightedProps> = ({ highlighted }: HighlightedProps) => (
+const Highlighted: React.FC<HighlightedProps> = ({
+    highlighted,
+    countryCode,
+}: HighlightedProps) => (
     <strong className={highlightWrapperStyles}>
         {' '}
         {highlighted.map((highlightText, idx) => (
             <span
                 key={idx}
                 className={highlightStyles}
-                dangerouslySetInnerHTML={{ __html: replacePlaceholders(highlightText) }}
+                dangerouslySetInnerHTML={{
+                    __html: replacePlaceholders(highlightText, countryCode),
+                }}
             />
         ))}
     </strong>
 );
 
-const EpicBody: React.FC<BodyProps> = ({ highlighted, paragraphs }: BodyProps) => (
+const EpicBody: React.FC<BodyProps> = ({ highlighted, paragraphs, countryCode }: BodyProps) => (
     <>
         {paragraphs.map((paragraph, idx) => (
             <p key={idx} className={bodyStyles}>
-                <span dangerouslySetInnerHTML={{ __html: replacePlaceholders(paragraph) }} />
+                <span
+                    dangerouslySetInnerHTML={{
+                        __html: replacePlaceholders(paragraph, countryCode),
+                    }}
+                />
 
                 {highlighted.length > 0 && idx === paragraphs.length - 1 && (
-                    <Highlighted highlighted={highlighted} />
+                    <Highlighted highlighted={highlighted} countryCode={countryCode} />
                 )}
             </p>
         ))}
     </>
 );
 
-export const ContributionsEpic: React.FC<Props> = ({ content, metadata }: Props) => {
+export const ContributionsEpic: React.FC<Props> = ({ content, metadata, countryCode }: Props) => {
```

Comment:
Why not bundle into `metadata`?

Reply:
I agree with Tom's logic here, I think it's better to keep localisation data separate from tracking data (what we now called metadata). We don't need to pass country code downstream to any functions whose responsibility is to generate a tracking link (and don't care about localisation).

Reply:
In any case, I've got a subsequent PR that refactors the component interface to group props in 4 separate objects: `content`, `tracking`, `localisation` and `targeting`. I suspect we'll be revisiting this soon.

Reply:
@andre1050 a nice thing about Typescript, is we can easily subset things as we go. I.e.

top-level `Metadata` is really just a general data structure.

Lower-level functions, can define (if they like) a subset relevant to them in their interface. E.g.

```
interface TrackingMetaData {
  ...
} // a subset of Metadata

const trackingLink = (meta: TrackingMetadata) => ...
```

It's generally a nice property to define interfaces at the point of use, rather than anticipate them/define them at the top-level:

https://blog.chewxy.com/2018/03/18/golang-interfaces/
https://twitter.com/davecheney/status/942593128355192832?lang=en

are some good sources on this approach.

Reply:
(To clarify, I'm not saying we should never split up `Metadata`, just it can be hard to guess the right structure up front, so it might be better to delay it and let the right structure 'emerge' by subsetting the interface as we go in specific packages/functions.

Reply:
Gotcha @nicl - I agree we should look into ways of subsetting metadata to be able to pass as much of it around as it's relevant to the components using it.
My preference is to merge this as is, there's a subsequent PR where we'll be able to have a better discussion around this as it splits all the data into the service into 4 objects - although as you say we might be able to merge this back together and subset the individual types.